src/main/steps/common/documents/upload/upload-your-documents/postController.ts

```diff
@@ -184,6 +186,10 @@ export default class UploadDocumentPostController extends PostController<AnyObje
       req.session.errors = handleError(req.session.errors, 'multipleFiles');
       return this.redirect(req, res, redirectUrl);
     }
+    if (caseData[documentDataRef].length >= maxLimit) {
```

Comment:
Just validate and handle errors in respective postcontrollers. But all the methods to validate file size should be placed in a common file and reused across postcontrollers

Reply:
@vivek-sekhar Made changes as per review comments and covered with unit tests also